gravitl/netclient
# Gra 985/host delete logic

**Should be reviewed after PR #134** merged
-> Host deletion logic through MQ

Testing Steps :- @theguy951357 

-> delete the host from UI, check locally whether the configuration is cleared for that server
-> you shouldn't receive any more updates through MQ from that server, you can verify with logs
